src/utils/index.ts

```diff
@@ -29,4 +29,33 @@ const copyToClipboard = async (text: string) => {
     });
 }
 
-export { showToast, hasError, copyToClipboard }
+const productIdentificationValue = (id: string, product: any) => {
```

Comment:
We could think of having helpers for different purposes, this could come under product